packages/exhibitor-migration-script/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
```

Comment:
these aren't needed in build scripts. All build scripts are always run with them set.